src/widgets/JudgeTerminal.ts

```diff
@@ -28,6 +28,28 @@ export class JudgeTerminal extends Panel {
     // Create toolbar
     const toolbar = new Widget();
 
+    // Reset to skeleton code button
+    const resetButton = document.createElement('button');
+    resetButton.className = 'jp-JudgeTerminal-resetButton';
+    refreshIcon.element({ container: resetButton });
+    const resetButtonLabel = document.createElement('span');
+    resetButtonLabel.className = 'jp-JudgeTerminal-resetButtonLabel';
+    resetButtonLabel.textContent = trans.__('Reset to skeleton code');
+    resetButton.addEventListener('click', async () => {
```

Comment:
await 없는데 굳이 async 쓰지 않아도 되지 않을까요?
```suggestion
    resetButton.addEventListener('click', () => {
```